test/test_helper.rb

```diff
@@ -43,3 +43,10 @@ def build_tweet
 end
 
 require 'mocha/setup'
+
+require 'vcr'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.